action.yml

```diff
@@ -153,4 +153,7 @@ runs:
           diff -B west.yml tmp.yml | patch west.yml - || true
           git add west.yml
           git commit --amend --no-edit
+          git fetch origin
```

Comment:
I believe origin points to forked repo here so I think you should add remote which points to real main, right?

Reply:
You are right. This needs to be fixed

Reply:
I looked through it, it seems correct: https://github.com/nrfconnect/action-manifest-pr/pull/7/files#diff-1243c5424efaaa19bd8e813c5e6f6da46316e63761421b3e5f5c8ced9a36e6b6R66 checks out the "real repo". The remote `fork` is used for the PR: https://github.com/nrfconnect/action-manifest-pr/pull/7/files#diff-1243c5424efaaa19bd8e813c5e6f6da46316e63761421b3e5f5c8ced9a36e6b6R122

Reply:
This was fixed.